app/front_assets/stylesheets/pages/internship_agreements.scss

```
@@ -0,0 +1,27 @@
+.internship-agreement{
+  .title{
+    padding: 10px 20px;
```

```suggestion
    padding: 10px 20px;
```

sur les padding, utiliser les classes d'helpers bootstrap . py-1 [padding-top/bottom: ?1?rem] px-2[padding-left/right: ?2rem?]. sinon on va perdre en consistance ac le temps

---

ok